src/tasks/WasmAppBuilder/WasmAppBuilder.csproj

```diff
@@ -29,6 +29,7 @@
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildTasksCoreVersion)" />
     <!-- we need to explicitly reference System.Reflection.MetadataLoadContext 8.0 or newer for function pointer support,
          because the default version pulled in from MSBuild is lower. -->
+    <!-- TODO: this can be removed once MSBuild brings in a new enough version -->
```

Comment:
I would remove the comment. S.R.MLC is a direct dependency now and it's not bad to reference direct dependencies directly.

Reply:
We only reference it directly to upgrade the transitive reference from Microsoft.Build. Once that is upgraded I don't see why we should be referencing it?